python/hsfs/engine/spark.py

```diff
@@ -53,10 +53,48 @@ def __init__(self):
         self._spark_session.conf.set("hive.exec.dynamic.partition.mode", "nonstrict")
         self._spark_session.conf.set("spark.sql.hive.convertMetastoreParquet", "false")
 
+        self.validate_spark_configuration()
+
         if importlib.util.find_spec("pydoop"):
             # If we are on Databricks don't setup Pydoop as it's not available and cannot be easily installed.
             util.setup_pydoop()
 
+    def validate_spark_configuration(self):
+        key = "spark.hadoop.hops.ssl.trustore.name"
+        if(not self._spark_session.conf.get(key)):
```

Comment:
no need for parenthesis in python:
```suggestion
        if not self._spark_session.conf.get(key):
```